client/site-profiler/components/metrics-menu.tsx

```diff
@@ -0,0 +1,71 @@
+import { Button } from '@automattic/components';
+import styled from '@emotion/styled';
+import { useTranslate } from 'i18n-calypso';
+import React, { useEffect, useState } from 'react';
+import SectionNav from 'calypso/components/section-nav';
+import NavItem from 'calypso/components/section-nav/item';
+import NavTabs from 'calypso/components/section-nav/tabs';
+import StickyPanel from 'calypso/components/sticky-panel';
+import { useIsMenuSectionVisible } from '../hooks/use-is-menu-section-visible';
+
+const FullReportButton = styled( Button )`
+	margin-right: 8px;
+	border-radius: 4px;
+	box-shadow: 0px 1px 2px 0px rgba( 0, 0, 0, 0.05 );
+`;
+
+const SectionNavbar = styled( SectionNav )`
+	margin-bottom: 32px;
+`;
+
+interface MetricsMenuProps {
+	basicMetricsRef?: React.RefObject< HTMLObjectElement >;
+}
+
+enum MetricsMenuTabs {
+	basic = 'basic',
+}
+
+export const MetricsMenu: React.FC< MetricsMenuProps > = ( props: MetricsMenuProps ) => {
+	const translate = useTranslate();
+	const { basicMetricsRef } = props;
+
+	const references = {
+		[ MetricsMenuTabs.basic ]: basicMetricsRef,
+	};
+
+	const [ selectedTab, setSelectedTab ] = useState< MetricsMenuTabs | undefined >();
+	const basicMetricsVisible = useIsMenuSectionVisible( basicMetricsRef );
+
+	useEffect( () => {
+		if ( basicMetricsVisible ) {
+			setSelectedTab( MetricsMenuTabs.basic );
+		} else {
+			setSelectedTab( undefined );
+		}
+	}, [ basicMetricsVisible ] );
+
+	const onMenuItemClick = ( tab: MetricsMenuTabs ) => {
+		if ( references[ tab ]?.current ) {
+			references[ tab ]?.current?.scrollIntoView( { behavior: 'smooth' } );
+		}
+	};
+
+	return (
+		<StickyPanel>
+			<SectionNavbar>
+				<NavTabs>
+					<NavItem
+						onClick={ () => onMenuItemClick( MetricsMenuTabs.basic ) }
+						selected={ selectedTab === MetricsMenuTabs.basic }
+					>
+						{ translate( 'Basic Performance Metrics' ) }
```

Comment:
Can we collect the nav-items dynamically? Or create a list with the items and title translations locally?

Reply:
I was planning to do this in the future when we had more than 1 item, but it was easy enough to add right now :)

https://github.com/Automattic/wp-calypso/pull/90404/commits/4400d05ae16d654ff87c522b111b65770f39166a